docs/web-apps/automated-testing/_partials/_advanced-playwright.md

```diff
@@ -43,4 +43,27 @@ You can use Playwright to test your app on any browser and emulate a real device
 
 :::note
 Playwright is not a real Emulator or Simulator. It just emulates the browser behavior such as `userAgent`, `screenSize`, `viewport` and etc.
-:::
\ No newline at end of file
+:::
+
+## Screenshots
+
+To enable Playwright screenshot capture, please refer to the [Playwright Config](https://playwright.dev/docs/test-configuration#automatic-screenshots).
```

Comment:
The anchor in this url (`automatic-screenshots`) doesn't exist in the page. Did you mean this url? https://playwright.dev/docs/test-use-options#recording-options

Reply:
Given that automatic screenshots won't work as expected with our reporter, we probably don't want to suggest that they enable this. The phrasing `To enable Playwright screenshot capture...` suggests this feature _will_ work on sauce.

Reply:
> The anchor in this url (`automatic-screenshots`) doesn't exist in the page. Did you mean this url? https://playwright.dev/docs/test-use-options#recording-options

yep. Weird. Yesterday the url seems worked. Will update it.